executor/builder.go

```diff
@@ -856,29 +862,16 @@ func (b *executorBuilder) buildTableReader(v *plan.PhysicalTableReader) Executor
 		ranges:    ts.Ranges,
 	}
 
-	for _, col := range v.Schema().Columns {
-		dagReq.OutputOffsets = append(dagReq.OutputOffsets, uint32(col.Index))
+	for i := range v.Schema().Columns {
+		dagReq.OutputOffsets = append(dagReq.OutputOffsets, uint32(i))
```

Comment:
It's hard to implement it now, because the indices of table reader's schema are wrong. We can make it right after moving resolve phase to physical plan.